src/packages/database/query/index.js

```diff
@@ -164,10 +165,10 @@ class Query<+T: any> extends Promise {
         this.snapshots = this.snapshots
           .filter(([method]) => method !== 'orderBy')
           .concat([
-            ['orderBy', [
-              `${this.model.tableName}.${columnName}`,
-              direction
-            ]]
+            ['orderByRaw', uniq([columnName, this.model.primaryKey])
+              .map(key => `${this.model.tableName}.${key}`)
+              .join(', ') + ` ${direction}`
```

Comment:
Looks like you need to add an `// eslint-disable-line prefer-template` comment to `L170`.